packages/bsky/src/api/blob-resolver.ts

```diff
@@ -31,42 +32,12 @@ export const createRouter = (ctx: AppContext): express.Router => {
         return next(createError(400, 'Invalid cid'))
       }
 
-      const [{ pds }, takedown] = await Promise.all([
-        ctx.didResolver.resolveAtpData(did), // @TODO cache did info
-        ctx.db.db
-          .selectFrom('moderation_action_subject_blob')
-          .innerJoin(
-            'moderation_action',
-            'moderation_action.id',
-            'moderation_action_subject_blob.actionId',
-          )
-          .where('cid', '=', cidStr)
-          .where('action', '=', TAKEDOWN)
-          .where('reversedAt', 'is', null)
-          .executeTakeFirst(),
-      ])
-      if (takedown) {
-        return next(createError(404, 'Blob not found'))
-      }
-
-      const blobResult = await retryHttp(() =>
-        getBlob({ pds, did, cid: cidStr }),
-      )
-
-      const imageStream: Readable = blobResult.data
-      const verifyCid = new VerifyCidTransform(cid)
+      const verifiedImage = await resolveBlob(did, cid, ctx)
 
-      // Send chunked response, destroying stream early (before
-      // closing chunk) if the bytes don't match the expected cid.
-      res.statusCode = 200
```

Comment:
did we end up dropping these headers/status code?

Reply:
Oops, really not sure how I did that 🙃 thanks for catching!